### PR TITLE
[TT-6587] Add GoDoc for Mock Response

### PR DESCRIPTION
**Pull request by furkansenharputlu**
This PR adds the missing GoDoc for mock response config.

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4325/merge
Commit:  
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3182477945)

**Comment by titpetric** [comment]
Please close this in favor of #4327 - that one adds full godoc comments to the complete oas package. If there are some content differences (I didn't copy paste), feel free to add your changes here into that PR as an additional comment.

File: apidef/oas/operation.go [hunk 1]
```diff
@@ -474,11 +474,16 @@ func (s *OAS) fillOASValidateRequest(metas []apidef.ValidateRequestMeta) {
...
-	FromOASExamples *FromOASExamples  `bson:"fromOASExamples,omitempty" json:"fromOASExamples,omitempty"`
+	// Enabled enables the mock response middleware.
+	Enabled bool `bson:"enabled" json:"enabled"`
+	// Code is the HTTP response code that will be returned.
+	Code int `bson:"code,omitempty" json:"code,omitempty"`
+	// Body is the HTTP response body that will be returned.
+	Body string `bson:"body,omitempty" json:"body,omitempty"`
+	// Headers are the HTTP response headers that will be returned.
+	Headers map[string]string `bson:"headers,omitempty" json:"headers,omitempty"`
+	// FromOASExamples is the configuration to extract a mock response from OAS documentation.
```
Comment (lghiur) [inline apidef/oas/operation.go:485]: where are the rest of fields from `fromExamples` docuemnted?

**Review by lghiur — APPROVED** [review]

File: apidef/oas/operation.go (hunk 1 above)
Comment (furkansenharputlu) [reply apidef/oas/operation.go:485]: They are merged in another pr. This one was missing.

**Review by marksou — APPROVED** [review]
LGTM

**Comment by furkansenharputlu** [comment]
@titpetric We said we are going to merge this one because it blocks entire story. In your PR, handling the conflict will be easy.

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4325/merge
Commit: f5fc0887576fd354e4a8b83ede3b69639c94f42f 
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3240681327)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: ae3913abd4c30ca8f3bd2148a29aafc61791966c Add GoDoc for Mock Response (#4325)
Triggered by: push (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3240785856)

**Pull request merged**